handlers/apps.go

```diff
@@ -88,9 +88,9 @@ func getAppMetrics(w http.ResponseWriter, r *http.Request, promSupplier promClie
 }
 
 // CustomDashboard is the API handler to fetch runtime metrics to be displayed, related to a single app
-func CustomDashboard() func(http.ResponseWriter, *http.Request) {
+func CustomDashboard(w http.ResponseWriter, r *http.Request) {
 	cfg := business.DashboardsConfig()
-	return khttp.DashboardHandler(cfg)
+	khttp.DashboardHandler(r.URL.Query(), mux.Vars(r), w, cfg)
```

Comment:
I wonder if passing a 

```
khttp.DashboardHandler(r, w, cfg)
```

It's a more clear approach.
If you pass the w, passing pair (r, w) sounds cleaner to me, you can extract URL.Query() and mux.Vars() from inside.

Reply:
That was my first approach, and it didn't worked well at some point, but I can't remember exactly what was the problem. Something related to "mux.Vars" not working inside the handler, because of a different version of the model used between kiali and k-charted, iirc.
My goal here was to pass "simple" data types that will never conflict in version between kiali and k-charted.

Reply:
Actually, I remember: the goal was to not have any dependency on gorilla/mux within k-charted